synapse/storage/deviceinbox.py

```diff
@@ -31,10 +31,10 @@ def __init__(self, hs):
         super(DeviceInboxStore, self).__init__(hs)
 
         self.register_background_index_update(
-            "device_inbox_stream_index",
-            index_name="device_inbox_stream_id_user_id",
-            table="device_inbox",
-            columns=["stream_id", "user_id"],
+            "device_lists_stream_idx",
+            index_name="device_lists_stream_user_id",
+            table="device_lists_stream",
+            columns=["user_id", "device_id"],
```

Comment:
Umm, What happened to the `(stream_id, user_id)` index?
Should the new index be called `device_lists_stream_user_id_device_id`?